pages/manage/productions/create.vue

```diff
@@ -111,6 +115,9 @@
       locale() {
         return this.$i18n.locale;
       },
+      heading1Variant() {
+        return Heading1Variant;
+      },
```

Comment:
If we were to keep this variant, would we need to put the enum object in a computed object? Can't we just access it directly in the template if we import it? 🤔 @geoffreydhuyvetters do you maybe know more about the technical details here?

Reply:
a constant imported in the script tag doesn't seem to be usable in the template unless you expose it via a method, data or computed or the last and maybe best option is to set them as part of the vue object by setting `this.HeadingVariant = HeadingVariant` in the `created` method

Reply:
This all feels quite complex for using a simple constant in a prop. Not sure if there's a better solution though, except for not using a constant in the first place and just using the hardcoded string 😕

In any case if we don't need the variant here anyway we can leave it out altogether and check what @geoffreydhuyvetters thinks once he's feeling better about how to approach these enums in other components.